### PR TITLE
qemu.md

### PR DESCRIPTION
**Pull request by Cvetomird91**
I added a page for the QEMU emulator. 

It usually is operated though CLI and opens a dialog, native to the OS's GUI, in which virtual machine 
displays are shown.

It may be useful for users, who need to create virtual instances on architectures different than x86 and x64, as it supports PowerPC, MIPS, ARM and quite few other CPU architectures.

Update:
It comes with separate commands for each CPU architecture, e.g. qemu-system-ppc64, qemu-system-i386, qemu-system-arm. This is the reason examples in this markdown file are involving separate commands. With qemu-img you can create a virtual HDD device for any of your virtual machines, e.g. on which you can install a particular OS. 

All of those binaries are usually bundled with the standard QEMU package tha you can install on a Debian/Ubuntu system with for example apt-get.

### CODE REVIEW

File: pages/common/qemu.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# qemu
+
+> Generic machine emulator and virtualizer.
+> Supports a large variety of CPU architectures.
+
+- Create disk image with a specific size (in gigabytes):
+
+`qemu-img create {{image_name.img}} {{gygabytes}}G`
```
Comment (waldyrious) [inline pages/common/qemu.md]: there's a typo here: "gygabytes" --> "gigabytes"

**Comment by waldyrious** [comment]
Looks great, but is there any chance it can be shortened a bit?

**Comment by Cvetomird91** [comment]
I shortened it with two options. Now they are six. 

I can remove the option to boot from physical bootable storage device
(e.g. in case the user would like to test a bootable USB medium he just created).

I considered to be useful though. :)

**Comment by waldyrious** [comment]
6 examples are ok, thanks! LGTM :+1:

**Comment by igorshubovych** [comment]
I do not know too much about qemu. Why do you have several commands on a single page: `qemu-system-i386`, `qemu-im`, etc?

**Comment by Cvetomird91** [comment]
I updated my initial message here.

**Comment by igorshubovych** [comment]
They are different commands. I suggest they should be listed in different pages.

**Comment by waldyrious** [comment]
I'm not so sure @igorshubovych. Unless each command is as complex and versatile as e.g. git subcommands, it would probably be a disservice to the user to force them to jump between tldr pages rather than presenting a unified summary. @Cvetomird91 what do you say?

**Comment by igorshubovych** [comment]
Well, at least `qemu-img` is quite different from `qemu-system`.

**Comment by Cvetomird91** [comment]
All qemu-system-* commands have almost identical options.

 I just made diffs by comparing the outputs of "qemu-system-mips -help", "qemu-system-i386 -help" and "qemu-system-x86_64 -help" and they have almost the same options. The only differences were options which are specific for the particular CPU architecture. 

I've selected those options because they are most commonly used tasks in the typical workflow with such a hypervisor and were the commands I most commonly looked up when getting into QEMU. 

Those commands include the following steps.

- create a virtual HDD
- fire up a virtual machine by booting installation media (e.g. live Linux USB)
- install a particular operating system on that HDD via the bootable media
- boot the newly installed OS on the virtual HDD

 Those of you that've used VirtualBox can agree that it is used most commonly for these purposes, but the steps are performed through a GUI when using VB, not through CLI. That's the part that can be confusing at the beginning when a person starts using QEMU. ;)

**Comment by igorshubovych** [comment]
What about qemu-img command?

**Comment by Cvetomird91** [comment]
I can create a separate page for it shortly, as system administrators most probably use it to manage Xen/KVM instance storage devices.

I think it's best to stick to the contributions guideline and make the page more suitable for beginners:

> When in doubt, keep new command-line users in mind. Err on the side of clarity rather than terseness.

**Comment by waldyrious** [comment]
That sounds like a good plan @Cvetomird91 and I agree with that reasoning. For me we can merge this as-is.

**Comment by igorshubovych** [comment]
Thank you!

**Pull request merged**